Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.m

```diff
@@ -148,13 +148,7 @@ - (BOOL)isDefaultVariant
 {
 	if ([self.variants count] == 1) {
 		BUYProductVariant *productVariant = [self.variants firstObject];
-		BUYOptionValue *optionValue = [productVariant.options anyObject];
-		NSString *defaultTitleString = @"Default Title";
-		NSString *defaultString = @"Default";
-		if ([productVariant.title isEqualToString:defaultTitleString] &&
-			([optionValue.value isEqualToString:defaultTitleString] || [optionValue.value isEqualToString:defaultString])) {
-			return YES;
-		}
+		return productVariant.isDefault;
```

Comment:
You wrote two different methods with same logic in different styles. Probably fine either way, but curious if you think one style is better than the other.